qrenderdoc/Windows/PipelineState/D3D12PipelineStateViewer.cpp

```diff
@@ -1169,7 +1169,9 @@ void D3D12PipelineStateViewer::setShaderState(const D3D12Pipe::Shader &stage, RD
           {
             bind = &bm;
             shaderCBuf = &res;
-            tag = QVariant::fromValue(D3D12CBufTag(i));
+            D3D12CBufTag cbufTag(i);
+            cbufTag.reg = reg - bm.bind;
+            tag = QVariant::fromValue(cbufTag);
```

Comment:
It's not important to conserve space in this struct - let's add a new member `arrayIdx` for clarity rather than re-using the `reg` member.

Reply:
Done